src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -6,7 +6,10 @@
                 v-translate="'selectedDataset'"
                 id="selectedDatasetSpan"
             ></span>
-            <a :href="selectedDataset.url" target="_blank">
+            <a v-if="releaseName" :href="selectedDataset.url + '/releases'" target="_blank">
```

Comment:
The url targets the releases page for the dataset rather than that specific release. I would need ADR to supply the release's specific url with the release if I were to target it directly as I cannot generate it from the information I have.

Reply:
URL is of the form https://dev.adr.fjelltopp.org/dataset/burundi-inputs-unaids-estimates-2021?activity_id=5c579cc1-3a14-4f89-b0a6-8813363940a0 and you should be able to get `activity_id` from the relevant `Release` object (you might just need to add that field to `Release` in `types.ts` in order to make it accessible).